sql-cli/examples/_dags/simple.py

```diff
@@ -2,11 +2,11 @@
 from airflow import DAG
 from airflow.utils import timezone
 from astro import sql as aql
-from astro.table import Table
+from astro.sql.table import Table
```

Comment:
Why this change?

Reply:
Because, in my opinion we should use the released python-sdk version - not the current one. Otherwise it will become very confusing for users if they see the example dag of an unreleased version. The import path change was introduced in #1013